lib/go/test/flow_dkg_test.go

```diff
@@ -547,3 +547,116 @@ func TestDKG(t *testing.T) {
 	})
 
 }
+
+func TestDKGNil(t *testing.T) {
+	b := newBlockchain()
+
+	env := templates.Environment{
+		FungibleTokenAddress: emulatorFTAddress,
+		FlowTokenAddress:     emulatorFlowTokenAddress,
+	}
+
+	accountKeys := test.AccountKeyGenerator()
+
+	// Create new keys for the DKG account and deploy
+	DKGAccountKey, DKGSigner := accountKeys.NewWithSigner()
+	DKGCode := contracts.FlowDKG()
+
+	DKGAddress, err := b.CreateAccount([]*flow.AccountKey{DKGAccountKey}, []sdktemplates.Contract{
+		{
+			Name:   "FlowDKG",
+			Source: string(DKGCode),
+		},
+	})
+	if !assert.NoError(t, err) {
+		t.Log(err.Error())
+	}
```

Comment:
```suggestion
	assert.NoError(t, err)
```
`NoError` will log the error itself if non-nil